manager/manager.py

```diff
@@ -598,7 +598,7 @@ def generate_dashboard(self, max_failures=MAX_FAILURES):
                 '<th class="hdr">Repo</th>'
                 '<th class="hdr">PreConvert</th>'
                 '<th class="hdr">Converted</th>'
-                '<th class="hdr">Destination</th>'
+                '<th class="hdr">Destination</th>',
```

Comment:
I agree with @phillip-hopper. This actually stumped me today when working on something else and seeing these multi-string call. Should use """ with multiple lines of text and then end with """,